src/Tasks/CultureInfoCache.cs

```diff
@@ -46,120 +60,286 @@ internal static bool IsValidCultureString(string name)
         {
             return ValidCultureNames.Contains(name);
         }
-        
+
 #if !FEATURE_CULTUREINFO_GETCULTURES
-        // copied from https://github.com/aspnet/Localization/blob/5e1fb16071affd15f15b9c732833f3ae2ac46e10/src/Microsoft.Framework.Globalization.CultureInfoCache/CultureInfoList.cs
-        // removed the empty string from the list
+        // Copied from https://github.com/aspnet/Localization/blob/5e1fb16071affd15f15b9c732833f3ae2ac46e10/src/Microsoft.Framework.Globalization.CultureInfoCache/CultureInfoList.cs
+        // Regenerated using the tool (removed by https://github.com/aspnet/Localization/pull/130)
+        //   * Removed the empty string from the list
+        //   * Any cultures not present when regenerated were retained
+        //   * Added the Windows pseudo-locales
```

Comment:
Should these be added to this list when they're going to be added above to handle the dynamic list-of-cultures case?

Reply:
When `HardcodedCultureNames` is used the method returns early on line 30. I can either rework the `#if !FEATURE_CULTUREINFO_GETCULTURES` to not return early and remove them from the hard-coded list, or leave it as-is.

Reply:
Ah, so it is. This is great, then.